google-codelab-step/BUILD

```diff
@@ -0,0 +1,46 @@
+load("//tools:defs.bzl",
+     "closure_js_library", "closure_js_binary", "closure_js_test")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_binary", "sass_library")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "google_codelab_step_files",
+    srcs = glob([
+        "*.html",
+        "*.png",
+    ]) + [
+        ":google_codelab_step_scss_bin",
+        ":google_codelab_step_bin",
+    ],
+)
+
+# Codelab step.
+closure_js_library(
+    name = "google_codelab_step",
+    srcs = ["google_codelab_step.js"],
+    convention = "GOOGLE",
+    deps = [
+        "@io_bazel_rules_closure//closure/library:library"
```

Comment:
A bit shorter dep is `@io_bazel_rules_closure//closure/library` because the `:library` suffix is default when none specified.